### PR TITLE
test double slash when input = '/'

### PR DESCRIPTION
**Pull request by ashmaroli**
test #5541

### CODE REVIEW

File: test/test_filters.rb [hunk 1]
```diff
@@ -368,6 +368,24 @@ def select; end
...
+        refute_equal "http://example.com/base//", filter.absolute_url(page_url)
+      end
+
+      should "not append a forward slash if input is '/' and nil 'baseurl'" do
+        page_url = "/"
+        filter = make_filter_mock({
+          "url"     => "http://example.com",
+          "baseurl" => nil
+        })
+        refute_equal "http://example.com//", filter.absolute_url(page_url)
```
Comment (pathawks) [inline test/test_filters.rb]: Same as above

File: test/test_filters.rb [hunk 2]
```diff
@@ -368,6 +368,24 @@ def select; end
         assert_equal "http://example.com/base", filter.absolute_url(page_url)
       end
 
+      should "not append a forward slash if input is '/'" do
+        page_url = "/"
+        filter = make_filter_mock({
+          "url"     => "http://example.com",
+          "baseurl" => "/base"
+        })
+        refute_equal "http://example.com/base//", filter.absolute_url(page_url)
```
Comment (pathawks) [inline test/test_filters.rb]: Can we `assert` that it is proper, rather than refuting this particular wrong case?

Comment (ashmaroli) [inline test/test_filters.rb]: done..

**Comment by parkr** [comment]
@benbalter, does this look good to you? I don't see a failing test in the history and I don't trust my reading well enough since I was the one who let this slip through in the first place. 😸

**Comment by pathawks** [comment]
Shouldn't these tests currently be failing? Was something else causing jekyll/jekyll-feed#145?

**Comment by ashmaroli** [comment]
I was not able to reproduce the error on my Windows machine. Hence wrote these tests to have Travis check it..
an edge case I didn't include in the tests : 
`test: if the filter produces "//" when the input and baseurl equals "/"`

**Comment by e0i** [comment]
I have applied the changes manually and checks didn't fail at all. Perhaps it was indeed something else that caused the situation described at #5541 as @pathawks mentioned?

**Comment by pathawks** [comment]
What's up with AppVeyor?

**Comment by ashmaroli** [comment]
> What's up with AppVeyor?

@pathawks the test log pre-dates fixing AppVeyor on the `master`. Restarting the build should sort this. To be on the safer side, I'll update the branch to latest `master` and initiate a new build altogether.

**Comment by parkr** [comment]
Always good to have extra tests for things like this.

@jekyllbot: merge +bug

**Comment by ashmaroli** [comment]
@parkr I think it would be better that @jekyllbot adds a single label `bug-fix` instead of the current two.. mainly because tagging a PR with `bug` doesn't seem that right.
